src/GoogleAuthService.ts

```diff
@@ -1,8 +1,8 @@
 import {Injectable} from "@angular/core";
 import {Observable} from "rxjs/Observable";
 import {Observer} from "rxjs/Observer";
-import 'rxjs/add/observable/of';
-import { mergeMap } from 'rxjs/operators';
+import { of } from 'rxjs';
```

Comment:
isnt it importing all rxjs ?

Reply:
No, in new version of RxJs this is the right way